client/test/unit/specs/components/JobList.spec.js

```diff
@@ -126,329 +118,76 @@ describe('Unit | Component | JobList.vue', () => {
         expect(jobTitles[0].textContent).to.equal('Today\'s mission');
-        expect(jobTitles[1].textContent).to.equal('Yesterday\'s mission');
-        expect(jobTitles[2].textContent).to.equal('Old mission');
+        expect(jobTitles[1].textContent).to.equal('Old mission');
+        expect(jobTitles[2].textContent).to.equal('Yesterday\'s mission');
         expect(jobTitles[3].textContent).to.equal('Very old mission');
       })));
     });
-
-
-    describe('after jobs are loaded, a country filter is applied', () => {
-      beforeEach(() => {
-        // given
-        expectedJobs = [
-          {
-            id: 2,
-            activity: {
-              title: 'Tech Lead mission 2',
-            },
-            project: {
-              name: 'Spark/Hive Archi for eCRM',
-              status: 'mission_signed',
-              customer: {
-                sector: {
-                  name: 'Australia',
-                },
-              },
-              business_contact: {
-                nickname: 'ABC',
-              },
-            },
-          },
-          {
-            id: 1,
-            activity: {
-              title: 'Tech Lead mission 1',
-            },
-            project: {
-              name: 'Système d\'excellence',
-              status: 'proposal_in_progress',
-              customer: {
-                sector: {
-                  name: 'FR - La Poste',
-                },
-              },
-              business_contact: {
-                nickname: 'ABC',
-              },
-            },
-          },
-        ];
-
-        sinon.stub(projectStatus, 'sort').returns(expectedJobs);
-        sinon.stub(jobsApi, 'fetchAll').resolves(jobFixture);
-
-        // when
-        component = new Constructor().$mount();
-      });
-
-      afterEach(() => {
-        projectStatus.sort.restore();
-        jobsApi.fetchAll.restore();
-      });
-
-      it('should render as many jobs as received from the API', () => Vue.nextTick().then(() => Vue.nextTick().then(() => {
-        expect(getJobCardsCount(component)).to.equal(2);
-      })));
-
-      describe('when selecting jobs in France and overseas', () => {
-        it('should display the two listed jobs', () => Vue.nextTick().then(() => {
-          // When
-          component.onSelectedCountryFilter('anyCountry');
-          // Then
-          return Vue.nextTick().then(() => {
-            expect(getJobCardsCount(component)).to.equal(2);
-          });
-        }));
-      });
-
-      describe('when selecting only jobs in France', () => {
-        it('should only display one job', () => Vue.nextTick().then(() => {
-          // When
-          component.onSelectedCountryFilter('France');
-          // Then
-          return Vue.nextTick().then(() => {
-            expect(getJobCardsCount(component)).to.equal(1);
-          });
-        }));
-      });
-
-      describe('when selecting only jobs in Australia', () => {
```

Comment:
:1st_place_medal: 
(J'allais te dire de supprimer parce que 1 describe par it c'était too much, et parce que c'était trop exhaustif)

Reply:
ce truc était hyper pourri. Je n'avais pas senti le truc lorsque j'avais relu le code, mais les computed property, c'est tellement plus fort !

Reply:
Ok